scipy/signal/_peak_finding.py

```diff
@@ -241,9 +241,9 @@ def argrelextrema(data, comparator, axis=0, order=1, mode='clip'):
     return np.where(results)
 
 
-def peak_prominences(vector, peaks, wlen=None):
+def peak_prominences(vector, peaks, wlen=None, ret_bases=False):
```

Comment:
can't you just use `array` instead of `vector` which is more of a physical concept that also needs directions so on?

Reply:
I used `vector` because I want to convey that the input is expected to be one-dimensional and  `find_peaks_cwt` already uses this name. `array` is so broad we could go with `x` as well which seems to be the SciPy standard anyway. `ts` for timeseries would be possible as well but I'm not a huge fan. I think if `vector` is not wanted we should go with `x`.

Actually I would prefer the name `signal` but that one is even worse considering the module name... I don't suppose we could change the package name to `scipy.dsp`? 😉

Reply:
+1 for `x` as we use that most other places